x/ibc/testing/chain_test.go

```diff
@@ -33,6 +33,7 @@ func TestCreateSortedSignerArray(t *testing.T) {
 
 	// smaller address
 	validator1.Address = []byte{1}
+	validator2.Address = []byte{2}
```

Comment:
I ran into indeterminacy with this

Reply:
what do you mean?

Reply:
I once ran the tests and they failed. I reran the tests and they passed. I didn't investigate too much further, you are welcome to take a deeper look